NEWS.md

```diff
@@ -36,15 +36,15 @@
 
 ### Dependencies
 * Add `java` `17`
-* Add `folio-spring-base` `6.0.1`
+* Add `folio-spring-base` `7.0.0`
```

Comment:
Isn't Dependencies for v1.0.0 release?
I think we should add it to v2.0.0 release

@psmagin?

Reply:
Reverted.